src/hash.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright (c) 2019, Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+//////////////////////////////////////////////////////////////////////////////
+
+#include <functional>
+#include <memory>
+#include <type_traits>
+
+#include "enums.h"
+#include "hash.h"
+#include "type.h"
+#include "utils.h"
+
+using namespace yarpgen;
+
+inline void Hash::hashCombine(size_t value) {
```

Comment:
Why is "inline" one-liner function in .cpp file?

Reply:
Removed